build/gulpfile.hygiene.js

```diff
@@ -109,7 +109,8 @@ const indentationFilter = [
 	'!extensions/sql-database-projects/src/test/baselines/*.sqlproj',
 	'!extensions/big-data-cluster/src/bigDataCluster/controller/apiGenerated.ts',
 	'!extensions/big-data-cluster/src/bigDataCluster/controller/clusterApiGenerated2.ts',
-	'!resources/linux/snap/electron-launch'
+	'!resources/linux/snap/electron-launch',
```

Comment:
You can't add stuff here to the bottom of the list, you need to add it to the middle of the list or you'll get conflicts like this.

Reply:
Well that's weird. Why? Is the distro merge adding stuff onto the end that it's conflicting with or something?

Reply:
Yeah the distro has stuff at the end of this. There's a cleanup we can do there to make sure this stops conflicting.

@anthonydresser do you think we can make a new array in distro and append it programmatically to this array? I've seen this issue come up a bunch of times now.